dcos/mesos.py

```diff
@@ -1246,7 +1245,15 @@ def _process_output_stream(self, response):
             self.print_output_event.wait()
 
         try:
-            for chunk in response.iter_content(chunk_size=None):
+            # Since upgrading to python 3.5 we now have to read from the
+            # response in chunk sizes of 1 byte in order to avoid blocking
+            # until the connection is closed. Once we upgrade python again, we
+            # should be able to remove this restriction and change it back to
+            # 'chunk_size=None'. See the links below for the bug reports.
+            #
+            # https://github.com/kennethreitz/requests/issues/3421
+            # https://github.com/shazow/urllib3/issues/928
+            for chunk in response.iter_content(chunk_size=1):
                 records = self.decoder.decode(chunk)
```

Comment:
@klueska I've left a comment on the JIRA about this. Let's try to track it down a little farther before we make this change